### PR TITLE
Bumping versions and updating the docs

### PR DESCRIPTION
**Pull request by lostintangent**
This PR makes the following changes:

1. Updates the CLI screenshot to include the latest command set (e.g. promote)
2. Adds a little more verbiage to the description of "access key add" and promote
3. Bumps the CLI's version to 1.1.0-beta, and updates its SDK dependency to point to 1.1.0-beta (and accept great minor/patch updates)
4. Bumps the management and acquisition SDK's version to 1.1.1-beta

### CODE REVIEW

File: cli/README.md [hunk 1]
```diff
@@ -43,16 +42,16 @@ code-push access-key ls
 code-push access-key rm <accessKey>
 ```
 
-You can generate additional access keys for your account with a description tagged to each key. This lets you log in to CodePush and manage your apps without going through the 3rd party authentication step, e.g. from a CI environment.
+If you need additional keys, which can be used in a "headless" environment (particularly CI), without needing to give acces to your GitHub and/or Microsoft crendentials, you can run the following command to create one (along with a description of what it is for):
```
Comment (dlebu) [inline cli/README.md]: typo: access

File: cli/README.md [hunk 2]
```diff
@@ -43,16 +42,16 @@ code-push access-key ls
...
 
 ```
 code-push access-key add "VSO Integration"
 ```
 
-After generating the access key, you can call the login command with the generated key.
+After creating the new key, you can specify it's value using the `--accessKey` flag of the `login` command, which allows you to perform the "headless" authentication, as opposed to launching a browser.
```
Comment (dlebu) [inline cli/README.md]: typo: its value

File: cli/README.md (hunk 1 above)
Comment (geof90) [inline cli/README.md]: Also this sentence feels kinda long.. maybe break it up "...crendentials. You can run..."

**Comment by geof90** [comment]
Small comment, LGTM otherwise.

File: cli/package.json [hunk 3]
```diff
@@ -20,10 +20,11 @@
     "react-native",
     "react"
   ],
+  "homepage": "https://microsoft.github.io/code-push",
   "author": "Microsoft Corporation",
   "license": "MIT",
   "dependencies": {
-    "code-push": "1.0.0-beta",
+    "code-push": "^1.1.0-beta",
```
Comment (dlebu) [inline cli/package.json]: Will the current changes in the CLI work with SDK version 1.1.0-beta? If there are breaking changes, we should not use caret ranges, but direct dependency on `1.1.1-beta`.  `^1.1.0-beta` means `>=1.1.0-beta` and `< 2.0.0`

Comment (dlebu) [inline cli/package.json]: Reading the [semver documentation](https://github.com/npm/node-semver#caret-ranges-123-025-004) confirms that the `1.1.1-beta` version would not even be allowed for dependencies specified with `^1.1.0-beta`. `1.1.0-beta.2` would be allowed though.

**Comment by dlebu** [comment]
LGTM!

**Pull request merged**